app/views/patients/update.js.erb

```diff
@@ -0,0 +1,37 @@
+// rerack approx gestation
+$('#patient_appointment_date')
+  .parent()
+  .find('span.help-block')
+  .text('Approx gestation at appt: <%= @patient.last_menstrual_period_at_appt %>')
+
+// rerack status
```

Comment:
Yeah. It's calling for the editing user on the audit trails still, so there's that n+1 we'll have to sort out, but I don't think that's terribly problematic at this stage (and iirc you've got an open PR that's a stab at the problem).